src/components/header/style.module.css

```diff
@@ -131,7 +131,6 @@
 
 			&:global(.home) {
 				text-transform: uppercase;
-				padding: 0 15px;
```

Comment:
Unrelated, but I found on a small enough screen (823px as an example) our header links wrap around:

![temp](https://github.com/preactjs/preact-www/assets/33403762/45ad4fbd-1d5a-4fba-9ac3-00d8ae82965c)

Notice 'REPL' blending into our logo. Removing the extra padding helps with this, and is unnoticeable.